src/com/biocollapse/model/Human.java

```diff
@@ -1,5 +1,162 @@
 package src.com.biocollapse.model;
 
+import src.com.biocollapse.util.GlobalRandom;
+
+enum GoalState {
+  to_home,
+  to_work,
+  to_hospital,
+}
+
 public class Human {
-  
+  private boolean infected;
+  private int infectedAt;
+  private boolean alive = true;
+  private boolean immune;
+  private MapPosition pos;
+  private MapPosition goalPos;
+  private GoalState goalState = GoalState.to_home;
+  private MapPosition workPos;
+  private MapPosition homePos;
+  // ! I dont know if its a good idea to store the config inside of every human
+  // ! We should change that later on! I just don't want to think about it right now.
+  private Config config;
```

Comment:
Yes i wouldnt store it in every human. We should think about making it a static class with a globally available Variable or a singleton. Something that makes it a global cache so you can get the values from everywhere in the code